postcss.config.js

```diff
@@ -2,20 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-var mapUrl = require("postcss-url-mapper");
```

Comment:
love seeing this cruft deleted

Reply:
You know you can always count on me to find the unused stuff 🔨

Reply:
there are stories in that code! 😉